backend/framework/qlf/dashboard/bokeh/qaskycont/main.py

```diff
@@ -37,14 +38,40 @@ def __init__(self, process_id, arm, spectrograph):
 
     def load_qa(self):
         cam = self.selected_arm+str(self.selected_spectrograph)
+        
+        # try:
```

Comment:
If you are not going to use it just delete it. We can recover code from git history